### ISSUE TITLE
Manifest populates main entry point with unrelated files with webpack 5

### ISSUE THREAD

**Issue by KrofDrakula** [shellscape/webpack-manifest-plugin#237]
- Manifest Plugin Version: 3.0.0
- Webpack Version: 5.6.0
- Operating System (or Browser): Chrome 87 / Win 10 64-bit
- Node Version: 12.16.1
- Link to reproduction _(⚠️ read below)_: https://repl.it/@KrofDrakula/manifest-plugin-repro

### Expected Behavior

When configuring external files via `file-loader` in the `rules` section, I would not expect the files to populate the manifest file with the entry name and the imported file. In the above example I would expect:

```json
{
  "main.js": "main-c53637abe0.js",
  "demo.svg": "demo-d96fd782eb31f2e1c11b1a3f0ab2bb18.svg"
}
```

### Actual Behavior

Importing an `.svg` file using `file-loader` with content hashes causes the manifest to override the entry name and appending the loaded file extension, in my case:

```json
{
  "main.js": "main-c53637abe0.js",
  "main.svg": "demo-d96fd782eb31f2e1c11b1a3f0ab2bb18.svg",
  "demo.svg": "demo-d96fd782eb31f2e1c11b1a3f0ab2bb18.svg"
}
```

### Additional Information

My guess is that this has to do with the plugin matching the content hash and assuming the loaded assets are also entry points.

**Comment by n1c01ash** [comment]
Having the same issue, using url loader to import fonts.

**Comment by shellscape** [comment]
@n1c01ash please refrain from "me too" replied. Use the reaction buttons instead.

**Comment by weaverryan** [comment]
Hi! I may have fixed this on #249 - there are details on how to test at the bottom :).
